provider/import.go

```diff
@@ -90,7 +90,14 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 		} else {
 			resources, err = p.Resources(ctx, t, f)
 			if err != nil {
-				return errors.WithStack(err)
+				// If the error is from the stdlib we just continue with them
+				// *errors.errorString is the standard lib
+				// *errors.fundamental is the github.com/pkg/errors
+				// This way if it's an std error or one from us we continue the import but we warn the user
+				if fmt.Sprintf("%T", err) == "*errors.errorString" || fmt.Sprintf("%T", err) == "*errors.fundamental" {
+					return errors.WithStack(err)
+				}
+				fmt.Fprintf(out, "\nunable to import resource %s: %s\n", t, err.Error())
 			}
```

Comment:
IDK if we should just pass all the errors or actually have the specific ones we want to ignore, at least for now I would actually check the error and ignore it if needed.

Also it may require more comparison, check `utils.Retry` for the full list, but as I said I think it would be better to have a specific comparison to the errors we want to ignore.

Reply:
We currently have a bunch of:
```go
return err
```
or
```go
return errors.Wrap(err, "...")
```

We don't have specifics `errcode`, making harder this filtering. Should we refactor the error handling to use more `errcode` and ease the error identification ?

Reply:
We could add an specific err code yep, when we use the sdk. But that should go on the specific provider (`aws` on this case).